oaknational/Oak-Web-Application
# Feat/curriculum data config

## Description

- adds appropriate config to hook us in to live curriculum data
- adds mocks and tests where appropriate

**TODO: add zod layer for this**

## Issue(s)

Fixes #1097 

## How to test

1. Go to {cloud link}
2. Navigate through from beta/teachers through to unit listings
3. You should see real data yo!


## Checklist
